tests/testthat/test-error-checks.R

```diff
@@ -145,6 +145,23 @@ test_that("isEmpty returns FALSE when objects are not empty", {
   expect_false(isEmpty(""))
 })
 
+test_that("hasEmptyString returns TRUE when strings are empty", {
+  expect_true(hasEmptyString(character(0)))
+  expect_true(hasEmptyString(c("", "y")))
+  expect_true(hasEmptyString(list("", "y")))
+  expect_true(hasEmptyString(c("", NA)))
+  expect_true(hasEmptyString(list("", NA)))
+  expect_true(hasEmptyString(NA))
+})
+
+test_that("hasEmptyString returns FALSE when string are not empty", {
+  expect_false(hasEmptyString("   abc   "))
+  expect_false(hasEmptyString(c("x", "y")))
+  expect_false(hasEmptyString(list("x", "y")))
+  expect_false(hasEmptyString(c(NA, "x", "y")))
+  expect_false(hasEmptyString(list(NA, "x", "y")))
```

Comment:
If `hasEmptyString(NA)` is `TRUE`, then `hasEmptyString(c(NA, "x", "y"))` should also be `TRUE`, otherwise it does not make sense to me.